linode/framework_provider_config.go

```diff
@@ -54,6 +54,13 @@ func (fp *FrameworkProvider) Configure(
 			return
 		}
 	}
+	if strings.HasPrefix(req.TerraformVersion, "0.") {
+		resp.Diagnostics.AddWarning(
+			"Support for Terraform 0.x is deprecated",
+			"Please upgrade to Terraform 1.x or later. "+
+				"Support for Terraform 0.x will be removed in a future version of the provider.",
```

Comment:
Super small nitpick:
```suggestion
				"Support for Terraform 0.x will be removed in a future version of this provider.",
```